core/backend/src/rpc/web/logging.ts

```diff
@@ -31,15 +31,19 @@ export class WebAppRpcLoggingBackend extends WebAppRpcLogging {
     const result = await invocation.result;
     const errorMessage = result.message ? result.message : result.objects; // Can be an error or an RpcSerializedValue
 
-    Logger.logInfo(CommonLoggerCategory.RpcInterfaceBackend, `${message}.${operationDescriptor}`, () => ({
+    const metadata = {
       method: invocation.request.method,
       path: invocation.request.path,
       status: invocation.status,
+      httpStatus: StatusCategory.for(result).code,
       errorMessage,
+      error: BentleyError.getErrorProps(result),
       // Alert! The following properties are required by Bentley DevOps standards. Do not change their names!
       ActivityId: invocation.request.id, // eslint-disable-line @typescript-eslint/naming-convention
       MachineName: this.getHostname(), // eslint-disable-line @typescript-eslint/naming-convention
       ...pathIds,
-    }));
+    };
+
+    Logger.logError(CommonLoggerCategory.RpcInterfaceBackend, `${message}.${operationDescriptor}`, metadata);
```

Comment:
This supposed to be logError?

Reply:
Yes.

It was `logInfo`, but this is only logged on error so I'm not sure why it was only `info` (+ this is inside function called `logErrorBackend`).